### PR TITLE
transition v3.0-RC2 to official v3.0

### PR DESCRIPTION
**Pull request by Alessandro100**
transitioned from v3.0-RC2 to official v3.0 🎉 #180 180
Screenshot 2024-04-11 at 14 46 28

### CODE REVIEW

File: gbfs-validator/versions/v3.0.js
Comment (Alessandro100) [inline gbfs-validator/versions/v3.0.js:1]: @richfab can you confirm v3.0-RC files required haven't changed for v3.0

Comment (richfab) [reply gbfs-validator/versions/v3.0.js:1]: I confirm, the file requirements are the same in v3.0 as in v3.0-RC.

Comment (richfab) [reply gbfs-validator/versions/v3.0.js:1]: @Alessandro100 I think `{ file: 'manifest', required: false }` should be added to `gbfs-validator/versions/v3.0.js`. 
It looks like it was missing in v3.0-RC.

**Review by richfab — APPROVED** [review]
Excellent work @Alessandro100! ⚡ 

I left one comment. Feel free to merge once you've resolve it 🚀

**Comment by richfab** [comment]
LGTM 🚀 You can merge @Alessandro100

**Pull request merged**